### PR TITLE
reworked and fix minimized drawer

### PR DESCRIPTION
**Pull request by AngelRionCervi**
Removed ugly text exceeding the border when minimizing the drawer and added icons and gatherer counts.

Thanks to pela who keeps 100go of ns2 images for the icons.

### CODE REVIEW

File: app/javascripts/components/main.js [hunk 1]
```diff
@@ -358,6 +358,7 @@ const GatherPage = React.createClass({
 	},
 
 	toggleCollapseMenu(e) {
+		console.log(this.state)
```
Comment (Absurdon) [inline app/javascripts/components/main.js]: could you remove this console.log please?

File: app/javascripts/components/gather.js [hunk 2]
```diff
@@ -1,1119 +1,1152 @@
...
+		return gatherArray.sort((a, b) => a.name - b.name);
+	},
+
+	drawerItemsClass(open){
+		if(!open){
+			return 'drawerItems'
+		}
+	},
+
+	getGatherTypeIcon(name){
```
Comment (Absurdon) [inline app/javascripts/components/gather.js]: You are tying the icons to the current set of gathers which contradicts the idea of having them easily configurable, could you weave this into the configuration maybe or have it as convention over configuration even?

File: app/javascripts/components/main.js (hunk 1 above)
Comment (AngelRionCervi) [reply app/javascripts/components/main.js]: Oops.

Removed.

File: app/javascripts/components/gather.js (hunk 2 above)
Comment (AngelRionCervi) [reply app/javascripts/components/gather.js]: Ok I should have explored the files more.

I removed the function to get the icons in gather.js and instead added the path in gather_pool.js.

Seems to work.

**Comment by Absurdon** [comment]
Sorry for letting this unanswered for so long but quite busy right now. 
Is this the final version now?

**Comment by AngelRionCervi** [comment]
No problem man ! If it's ok for you and you don't have any more recommendations, I guess it should be good to push.

**Comment by Absurdon** [comment]
merged and published. thanks for the effort

**Pull request merged**